src/Mod/Measure/Gui/TaskMeasure.h

```diff
@@ -65,15 +65,13 @@ class TaskMeasure: public TaskView::TaskDialog, public Gui::SelectionObserver
     bool hasSelection();
     void clearSelection();
     bool eventFilter(QObject* obj, QEvent* event) override;
-    void setMeasureObject(Measure::MeasureBase* obj);
```

Comment:
TaskMeasure.h lacks of a file guard. Please add one as otherwise compilation can fail due to duplicated class declarations.